randovania/games/cave_story/logic_database/Ruined Egg Corridor.txt

```diff
@@ -21,8 +21,12 @@ Extra - starting_script: <TRA0049:0099:0007:0006
       Trivial
   > Pickup (Dragon Chest)
       Any of the following:
-          Pacifist Strats (Advanced) or Has Weapon
+          # Kill dragon
+          Has Weapon
+          # Walk through the dragon
           Damage Boosting (Beginner) and Pacifist Strats (Intermediate) and Normal Damage ≥ 10
+          # Jump over dragon
```

Comment:
No `TODO: vid` for these?

Reply:
the other TODOs like this are for a reasonably difficult gravity hop trick that needs a vid with input display to be considered documented. this one is just a normal jump that's trivial to perform, just punishing if you can't tank a hit